packages/eslint-config-airbnb/rules/react.js

```diff
@@ -8,12 +8,22 @@ module.exports = {
   'rules': {
     // Prevent missing displayName in a React component definition
     'react/display-name': 0,
+    // Forbid certain propTypes (any, array, object)
+    'react/forbid-prop-types': 0,
```

Comment:
or would you rather me add a link pointing to the docs by each config, like this.
```
    // Forbid certain propTypes (any, array, object)
    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/forbid-prop-types.md
    'react/forbid-prop-types': 0,
```